gradle/libs.versions.toml

```diff
@@ -7,20 +7,23 @@ s3-mock = "2.11.0"
 retrofit = "2.9.0"
 gson = "2.10.1"
 protobuf = "3.25.1"
+okhttp-bom = "4.12.0"
 
 [libraries]
 kotlin-bom = { group = "org.jetbrains.kotlin", name = "kotlin-bom", version.ref = "kotlin" }
 kotlin-stdlib = { group = "org.jetbrains.kotlin", name = "kotlin-stdlib-jdk8", version.ref = "kotlin" }
 amazon-bom = { group = "software.amazon.awssdk", name = "bom", version.ref = "amazon-bom" }
-amazon-s3 = { group = "software.amazon.awssdk", name = "s3", version.ref = "amazon-bom" }
-amazon-sso = { group = "software.amazon.awssdk", name = "sso", version.ref = "amazon-bom" }
-amazon-sts = { group = "software.amazon.awssdk", name = "sts", version.ref = "amazon-bom" }
+amazon-s3 = { group = "software.amazon.awssdk", name = "s3" }
+amazon-sso = { group = "software.amazon.awssdk", name = "sso" }
+amazon-sts = { group = "software.amazon.awssdk", name = "sts" }
 adobe-s3-mock = { group = "com.adobe.testing", name = "s3mock", version.ref = "s3-mock" }
 google-cloud-storage = { group = "com.google.cloud", name = "google-cloud-storage", version.ref = "google-cloud-storage" }
 google-gson = { group = "com.google.code.gson", name = "gson", version.ref = "gson" }
 google-protobuf-java = { module = "com.google.protobuf:protobuf-java", version.ref = "protobuf" }
 retrofit-core = { module = "com.squareup.retrofit2:retrofit", version.ref = "retrofit" }
 retrofit-converter-gson = { module = "com.squareup.retrofit2:converter-gson", version.ref = "retrofit" }
+okhttp-bom = { group = "com.squareup.okhttp3", name = "okhttp-bom", version.ref = "okhttp-bom" }
```

Comment:
Added the entire bom so it can act as a dependency constraint for other okhttp sub-libs as well?